redaxo/src/addons/structure/lang/de_de.lang

```diff
@@ -1,7 +1,7 @@
 structure = Struktur
 
 title_structure = Strukturverwaltung
-homepage = Homepage
+website = Website
```

Comment:
@tbaddade Hab mir nochmal das Issue durchgelesen. Warum hast du dich eigentlich für "Website" und gegen "Hauptebene" entschieden?
Passt Hauptebene nicht noch besser, gerade auch bei Multidomain?

Reply:
> Warum hast du dich eigentlich für "Website" und gegen "Hauptebene" entschieden?

Hatte ich sogar zuerst. Aber das würde dann doppelt dastehen.

![screenshot 2018-05-16 17 36 46](https://user-images.githubusercontent.com/338267/40127489-c1534c0e-592f-11e8-9cdf-871bf7c4ea41.png)

Wenn wir das bisherige Wort ` / Hauptebene` dort löschen fände ich das insg. dann  stimmiger.

Info: das `/ Hauptebene` wird in der Tiefe dann mit dem Pfad ersetzt.

Reply:
Dann können wir auch den Header ändern in `Liste der Kategorien in der Hauptebene`

Reply:
Klingt doch nach einem Plan. :) 
Fände ich gut. 👍

Reply:
:+1: